### DISCUSSION TITLE
Issue with dynamic post colourpicker for page background

### DISCUSSION THREAD

**Discussion by CarlHodges** [elementor/elementor#23080] [Developers Community]
Hello all,
Anyone had issues with dynamic colour picker option for page background. Its all working for say colour picking text colours dynamically, however the page background colour, whilst the option is there, it seems to do nothing? Could be a bug or a fix required? unless I am missing something?

Fully updated and using Dev Builder.  Elementor Pro user.

**Comment by susanhope** [comment]
Hello @CarlHodges 

We only just started building this small community, not many people know about it YET! To help you out, we will be sharing a post which will include a link to this post in our main (huge) community on Facebook.
